packages/LOLBAS/PKGBUILD

```diff
@@ -0,0 +1,24 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname='LOLBAS'
```

Comment:
Change the name to `lolbas` (not capital). If you have it all in capital, the package name will be in capital.